docs/content/posts/2023-05-19-release-1.17.1.md

```diff
@@ -0,0 +1,195 @@
+---
+title:  "Apache Flink 1.17.1 Release Announcement"
+date: "2023-05-19T13:00:00.000Z"
+aliases:
+- /news/2023/05/19/release-1.17.1.html
+authors:
+- reswqa:
+  name: "Weijie Guo"
+  twitter: "WeijieGuo12"
+---
+
+The Apache Flink Community is pleased to announce the first bug fix release of the Flink 1.17 series.
+
+This release includes 75 bug fixes, vulnerability fixes, and minor improvements for Flink 1.17.
```

Comment:
NIT: I was wondering which number should be used in this case, since only 61 tasks are listed below.

Reply:
Any other concerns about this?